doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
If this is the new OVA instructions, then this PR must also include an update to 
https://github.com/Seagate/cortx/blob/main/QUICK_START.md
to update the OVA link in that page.

Reply:
done, it has been fixed